NOLF2/Shared/SaveLoadMgr.cpp

```diff
@@ -193,6 +193,23 @@ void CSaveLoadMgr::WriteContinueINI( char const* pszSaveKey, char const* pszSave
 	return;
 }
 
+#ifdef __LINUX
+static void CleanPathForLinux(char** path)
+{
+	char* ret = *path;
+	char* mod = ret;
+	while (*mod)
+	{
+		if (*mod == '\\')
+		{
+			*mod = '/';
+		}
+		mod++;
+	}
+
+	*path = ret;
```

Comment:
seems to be unneeded to assign it back to path.
looks to update the path in-place.

Reply:
Updated.